TemplateBundle/content/Application/QuickStart.App/BlankPage.fs

```diff
@@ -0,0 +1,117 @@
+namespace QuickStart.App
+
+/// This is a basic module
```

Comment:
I don't get it 😅

Reply:
I tried to explain the interaction between child components on a page I'll try to come up with something more clear